src/gc/impl/conservative/gc.d

```diff
@@ -259,12 +259,12 @@ class ConservativeGC : GC
 
     import core.internal.spinlock;
     static gcLock = shared(AlignedSpinLock)(SpinLock.Contention.lengthy);
-    static bool _inFinalizer;
+    static bool _isLocked;
```

Comment:
That's easily confusable with the global GC lock? Also it's purpose is not locking but detecting reentrancy from finalizers. I think the name is rather appropriate.